Presentation/presentation.md

```diff
@@ -0,0 +1,31 @@
+# Strefy
+![strefy](pictures/strefy.png)
```

Comment:
add this 
![image](https://github.com/knmlprz/safe-luggage/assets/71179061/10d28cbf-e2cf-429d-b7c6-dc27a140907c)